### PR TITLE
Changes leaderboard BabyLM

### PR DESCRIPTION
**Pull request by Ciroye**

### CODE REVIEW

File: backend/app/domain/services/builder_and_evaluation/eval_utils/metrics.py [hunk 1]
```diff
@@ -188,8 +188,17 @@ def get_accuracy_meta(task=None):
...
+        "unit": "%",
+        "pretty_name": "Performance",
+        "utility_direction": 1,
+        "offset": 0,
+    }
+
+
 def get_matthews_correlation_meta(task=None):
-    return {"unit": "%", "pretty_name": "Accuracy", "utility_direction": 1, "offset": 0}
+    return {"unit": "%", "pretty_name": "Matthews", "utility_direction": 1, "offset": 0}
```
Comment (remg1997) [inline backend/app/domain/services/builder_and_evaluation/eval_utils/metrics.py:201]: We probably want to rename this to Matthews's Correlation.

File: backend/app/domain/services/builder_and_evaluation/eval_utils/metrics.py [hunk 2]
```diff
@@ -236,6 +245,14 @@ def get_macro_f1_meta(task=None):
     return {"unit": "%", "pretty_name": "Macro F1", "utility_direction": 1, "offset": 0}
 
 
+def get_f1(predictions: list, targets: list):
+    return f1_score(targets, predictions, average="micro")
```
Comment (remg1997) [inline backend/app/domain/services/builder_and_evaluation/eval_utils/metrics.py:249]: We discussed Macro, right?

File: frontends/web/src/components/TaskLeaderboard/TaskModelLeaderboardCardWrapper.jsx [hunk 3]
```diff
@@ -147,7 +140,7 @@ export const TaskModelDefaultLeaderboard = taskModelLeaderboardCardWrapper(
   (task, api, setWeightsCallback) => {
     const metricIdToDataObj = {};
     const datasetIdToDataObj = {};
-
+    console.log("metrics", task);
```
Comment (remg1997) [inline frontends/web/src/components/TaskLeaderboard/TaskModelLeaderboardCardWrapper.jsx:143]: Keeping the log for a reason?

Comment (Ciroye) [reply frontends/web/src/components/TaskLeaderboard/TaskModelLeaderboardCardWrapper.jsx]: nope

File: frontends/web/src/new_front/context/CreateInterface/Context.tsx [hunk 4]
```diff
@@ -23,6 +23,8 @@ export const CreateInterfaceProvider: FC<CreateInterfaceProviderProps> = ({
       });
     } else {
       setMetadataExample((prevModelInputs) => {
+        console.log("input", input);
```
[the remark this answers is not in the document]
Comment (Ciroye) [reply frontends/web/src/new_front/context/CreateInterface/Context.tsx]: nope

**Review by remg1997 — APPROVED** [review]

**Pull request merged**